runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankTagsConfig.java

```diff
@@ -53,6 +55,40 @@ default boolean rememberTab()
 		return true;
 	}
 
+	@ConfigItem(
+		keyName = "displayQuantity",
+		name = "Display Quantity",
+		description = "Display an overlay of the quantity of items for the tag.",
+		position = 3
+	)
+	default boolean displayQuantity()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "autoSetQuantity",
+		name = "Auto set Quantity",
+		description = "Automatically set quantities when tagging your inventory.",
+		position = 4
+	)
+	default boolean autoSetQuantity()
+	{
+		return true;
+	}
+
+	@Alpha
```

Comment:
For configs, we list other annotations after the `@ConfigItem`, so this would become:

```java
@ConfigItem(
	// ...
)
@Alpha
```